src/zc/buildout/buildout.txt

```diff
@@ -1264,10 +1264,10 @@ Expressions must not contain either the ``#`` or the ``;`` character.
 Adding and removing options
 ---------------------------
 
-We can append and remove values to an option by using the + and -
+We can append and remove values to an option by using the ``+`` and ``-``
 operators.
 
-This is illustrated below; first we define a base configuration.
+This is illustrated below; first we define a base configuration::
```

Comment:
Changing this to ``::`` isn't strictly necessary since the example shown is a doctest and is detected properly.